packages/plugin-dialog/src/ui/DialogZoneDomRenderer.ts

```diff
@@ -0,0 +1,43 @@
+import { DomRenderingEngine } from '../../../plugin-dom/src/DomRenderingEngine';
+import { AbstractRenderer } from '../../../plugin-renderer/src/AbstractRenderer';
+import { DialogZoneNode } from './DialogZoneNode';
+import { VNode } from '../../../core/src/VNodes/VNode';
+import template from './Dialog.xml';
+import './Dialog.css';
+
+const container = document.createElement('jw-container');
+container.innerHTML = template;
+const floatItem = container.firstElementChild;
```

Comment:
Isn't it possible to assign `outerHTML` and use the element directly rather than having to resort to use its first child ?

Reply:
Nop. We can use xml parser but it's longer